pkg/agent/util/net_linux.go

```diff
@@ -290,6 +290,15 @@ func RenameInterface(from, to string) error {
 	if pollErr != nil {
 		return fmt.Errorf("failed to rename host interface name %s to %s", from, to)
 	}
+	// Fix for the issue https://github.com/antrea-io/antrea/issues/6301.
+	// In some new Linux versions which support AltName, if there is only one valid value in AlternativeNamesPolicy
+	// which is also occupied by an interface name, after the interface name changed from generated value to another one,
+	// the interface altname will be set to previous name immediately.
+	// This altname must be removed to avoid unexpected conflict.
+	if err := removeInterfaceAltName(to, from); err != nil {
+		klog.ErrorS(err, "Failed to remove AltName after interface renamed")
+		return fmt.Errorf("failed to remove AltName %s on interface %s", from, to)
```

Comment:
```suggestion
		return fmt.Errorf("failed to remove AltName %s on interface %s: %w", from, to, err)
```
don't handle the same error twice

Reply:
Fixed. Thanks.